src/renderer/utils/cn.ts

```diff
@@ -1,4 +1,4 @@
-import { type ClassValue, clsx } from 'clsx/lite';
+import { type ClassValue, clsx } from 'clsx';
```

Comment:
Did you find a reason for needing the non lite one? I had set that one because we didn't need the base one.

Reply:
No use-case, but was having issues with the new tsconfig and importing clsx/lite.